client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -92,3 +92,27 @@ def test_model_version_list_builds(
         sorted(build_dicts, key=lambda d: d["id"]),
     ):
         assert_build_fields(build, build_dict)
+
+
+@given(build_dict=build_dict())
+def test_build_start_scan_not_external_exception(mock_conn, build_dict):
+    """Test that start_scan() raises the expected error when the `external` parameter is not
+    included or False.  This can be removed when we support internal scans.
+    """
+    build = Build(mock_conn, build_dict)
+    with pytest.raises(
+        NotImplementedError,
+        match=(
+            "internal scans are not yet supported.  Please use `external=True`"
+            " parameter"
+        ),
```

Comment:
Did you re-run tests before re-requesting review? This doesn't match the exception message, and I'd expect this test to fail.

Reply:
Oh wow it looks like it passes 🤔

Reply:
Am...am I losing it? 😵‍💫

Reply:
I fixed up the trailing period, but missed the semi-colon change in the middle.  Just reran tests on my end and it did fail on that until I fixed it.  Sorry for the mixup.  Should be fixed now.